### PR TITLE
Add draft for Release 5.1

### PR DESCRIPTION
**Pull request by Siedlerchr**

### CODE REVIEW

File: _posts/2020-08-31-JabRef5-1..md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+For LaTeX users, JaRef now supports the entry types and fields from `biblatex-software` out of the box.
+
+[And many more! In total, there are over 100 changes](https://github.com/JabRef/jabref/blob/master/CHANGELOG.md) since our last offical release!
+
+**You can get it from [FOSShub](https://www.fosshub.com/JabRef.html).**
+
+Please report any other issues that you encounter in our [bug tracker](https://github.com/JabRef/jabref/issues) (in case it was not already reported before).
+
+## Breaking Changes
```
Comment (tobiasdiez) [inline _posts/2020-08-31-JabRef5-1..md:32]: This section can be removed, right? I'm not aware of any breaking changes from 5.0 to 5.1.

Comment (koppor) [reply _posts/2020-08-31-JabRef5-1..md]: Removed it. The groups format change happened long time ago.

**Comment by calixtus** [comment]
I changed some typos, made some stylistic changes and reordered it a little bit ("Despite some other changes" was twice) and added a few words about our free time :wink:

File: _posts/2020-08-31-JabRef5-1..md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+For LaTeX users, JaRef now supports the entry types and fields of [`biblatex-software`](https://ctan.org/pkg/biblatex-software) out of the box.
+
+Additionally to these improvements under the hood, we also tweaked the user interface:
+
+We added tooltips for nearly all fields in the [entry editor](https://docs.jabref.org/advanced/entryeditor) to show a description of the fields purpose. The comment field especially now supports markdown.
+A progress indicator in the toolbar provides an overview of running background tasks and shows their average progress. Also the toolbar should now scale more nicely in smaller windows.
+The preferences dialog got some smaller visual improvements and the structure was a bit refined.
+A major change affects users of Mac computers running Mac OS Catalina. You will no longer get any warning about insecure software. JabRef for Mac is now completely signed and certified. We also took the opportunity to visually adjust the installer.
+
+![JabRef 5.1 DMG screen](/img/jabref-5-1-macDmg.png)
```
Comment (tobiasdiez) [inline _posts/2020-08-31-JabRef5-1..md:26]: I don't think that this update is important enough to be the only picture in the release notes. What about the download on import and the progress indicator instead?

Comment (Siedlerchr) [reply _posts/2020-08-31-JabRef5-1..md:24]: That was my suggestion as well. We should add some more screenshots and maybe also from the preferennces

**Comment by calixtus** [comment]
Wouldn't say that prefs have changed much visually since 5.0

File: _posts/2020-08-31-JabRef5-1..md (hunk 2 above)
Comment (koppor) [reply _posts/2020-08-31-JabRef5-1..md:28]: We have the markdown file now. Is it OK to have both or should we remove this one?

**Pull request merged**